example/src/main/scala/SocketEchoServer.scala

```diff
@@ -9,6 +9,8 @@ object SocketEchoServer extends App {
   private val socket =
     Socket.forall[WebSocketFrame](msg => ZStream.repeat(msg).schedule(Schedule.spaced(1 second)).take(10))
 
+  implicit val httpNothingPartial: CanSupportPartial[Request, Nothing] = (_: Request) => Nil.head
```

Comment:
Is this needed?

Reply:
without it I got the following when doing the scala3 compile
```
[error] -- Error: /Users/brown/git/zio-http/example/src/main/scala/SocketEchoServer.scala:18:5
[error] 18 |    }
[error]    |     ^
[error]    |no implicit argument of type zhttp.http.CanSupportPartial[zhttp.http.Request, Nothing] was found for parameter error of method apply in class MakeCollectM
```